spec/controllers/your_results_controller_spec.rb

```diff
@@ -1,75 +1,100 @@
 RSpec.describe Wpcc::YourResultsController do
   routes { Wpcc::Engine.routes }
 
-  describe 'GET /' do
-    let(:contributions_calendar) { double('ContributionsCalendar') }
-    let(:session) do
-      {
-        eligible_salary: 12_124,
-        employee_percent: 1,
-        employer_percent: 1,
-        salary_frequency: 'week'
-      }
-    end
-    let(:schedule) { [period_contribution, period_contribution] }
-    let(:period_contribution) { double(Wpcc::PeriodContribution) }
-    let(:presenter) { double(Wpcc::PeriodContributionPresenter) }
-    let(:salary_message) { double(Wpcc::SalaryMessage) }
-    let(:salary_frequency) { double(Wpcc::SalaryFrequency) }
-
-    it 'schedules a contribution calendar with the session salary_frequency' do
-      args = session.merge(salary_frequency: 52)
+  let(:contributions_calendar) { double('ContributionsCalendar') }
+  let(:schedule) { [period_contribution, period_contribution] }
+  let(:period_contribution) { double(Wpcc::PeriodContribution) }
+  let(:period_percents) { [pcc, pcc] }
+  let(:pc_presenter) { double(Wpcc::PeriodContributionPresenter) }
+  let(:message_presenter) { double(Wpcc::MessagePresenter) }
+  let(:salary_message) { double(Wpcc::SalaryMessage) }
+  let(:salary_frequency) { double(Wpcc::SalaryFrequency) }
+  let(:legal_periods) { [legal_period, legal_period] }
+  let(:legal_period) { double(Wpcc::LegalPeriod) }
+  let(:legal_period_presenters) { [legal_period_presenter, legal_period_presenter] }
+  let(:legal_period_presenter) { double(Wpcc::LegalPeriodPresenter) }
+  # let(:view_context) { ActionController::Base.new.view_context }
```

Comment:
@margOnline should this be removed if it's commented out? Is it needed any longer?